src/http_server/api/v1/metrics.c

```diff
@@ -139,6 +139,38 @@ static void cb_mq_metrics(mk_mq_t *queue, void *data, size_t size)
     cleanup_metrics();
 }
 
+int string_cmp(const void* a_arg, const void* b_arg) {
+  char* a = *(char **)a_arg;
+  char* b = *(char **)b_arg;
+  return strcmp(a, b);
+}
+
+size_t extract_metric_name_end_position(char *s) {
+    int len = strlen(s);
```

Comment:
this is not necessary. If you look from the caller, what you are really passing as a reference is a flb_sds_t string, so you can use flb_sds_len(s) and avoid a strlen().

Reply:
Thanks, done.